### PR TITLE
[READY FOR REVIEW] Azure ML API new version 2020-03-01 with private link properties

### PR DESCRIPTION
**Pull request by guanyu-240**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by sanjaiganesh** [comment]
>         "actionsRequired": {

I would recommend leaving this as-is and not renaming it..  
Now the service has to ensure backward compatibility, between this and older api versions to do conversion.

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json:3146 in f9aa6fe. [](commit_id = f9aa6fe46cb67a5aaefb6990618f4ac742512e85, deletion_comment = False)

**Comment by sanjaiganesh** [comment]
>           "description": "Indicates whether the connection has been Approved/Rejected/Removed by the owner of the service."

nit- please review and mark as read-only, as appropriate. 

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json:3263 in 9983b7f. [](commit_id = 9983b7f4dea3561f0840823511dd3bf5a64fc0a6, deletion_comment = False)

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                "groupId": "Sql",
+                "requestMessage": "Please approve",
+                "status": "Approved"
+              }
+            }
+          ]
+        }
+      }
+    },
+    "202": {
```
Comment (zhaomuzhi) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json]: I believe 202 doesn't return response body

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                "groupId": "Sql",
+                "requestMessage": "Please approve",
+                "status": "Approved"
+              }
+            }
+          ]
+        }
+      }
+    },
+    "201": {
```
Comment (zhaomuzhi) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json]: also, 201 shouldn't return response body

**Review by zhaomuzhi — CHANGES_REQUESTED** [review]

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json (hunk 2 above)
Comment (guanyu-240) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json]: I think 201 should still return body but 202 shoudn't

**Comment by qianwens** [comment]
>         "actionsRequired": {

Change property name is a breaking change which we should avoid

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json:3142 in 09f8e19. [](commit_id = 09f8e1999f8f63a617672c5e5a5a721d3fceecd9, deletion_comment = False)

**Review by zhaomuzhi — APPROVED** [review]

**Review by qianwens — APPROVED** [review]

**Pull request merged**